package.json

```diff
@@ -3,7 +3,7 @@
   "author": "Microsoft Application Insights Team",
   "license": "MIT",
   "bugs": "https://github.com/microsoft/ApplicationInsights-node.js/issues",
-  "version": "3.0.0-beta.8",
+  "version": "3.0.0-beta.9",
```

Comment:
Are we not setting AZURE_MONITOR_DISTRO_VERSION with this? we should because there will be no way to differentiate data coming from this package in Statsbeat

Reply:
Updated the constant